install.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+# Navigate to user home directory
+sys_user="mob_app_usr"
+sys_user_dir=/home/$sys_user
+
+cd $sys_user_dir
+echo -e "Current directory: $(pwd)\n"
+
+# Check if Mobalysis repo exists - if yes, delete it
+if [[ -d "./Mobalysis" ]]
+then
+    echo -e "Mobalysis folder exists in current directory. Deleting it..."
+    rm -rf ./Mobalysis
+    echo -e "Deletion complete.\n"
+fi
+
+# Clone Mobalysis repo
+sudo git clone https://github.com/PatProMath/Mobalysis
+
+# Switch from current user to sys_user -> because we want to run the commands to set the environment variables as the sys_user
+# Altenative: use command below for each line to set variable
+# sudo su --login --command 'echo "export DBNAME=mobalytics" >> ~/.bashrc' $sys_user 
+sudo -u $sys_user bash << EOF
```

Comment:
Hi @Cyril-A. For the code below, I changed to a different user to set up the environment variables, then swithc back to original user. I was wondering if the script should just keep the `mob_app_usr` active?